recipes/gz-rendering/activate.sh

```diff
@@ -0,0 +1,3 @@
+export OGRE_RESOURCE_PATH=$CONDA_PREFIX/lib/OGRE
```

Comment:
At some point we could consider simplifying this by adding a JSON file to `$PREFIX/etc/conda/env_vars.d/gz-rendering.json` that contains a JSON dictionary with the env vars. Unfortunately this functionality will only be available with the next release of micromamba (already exists in conda) :)

Reply:
Ah cool, so this could be substitute multisheller scripts?

Reply:
Only for environment variables, yep.